core/src/test/scala/integration/kafka/server/DynamicBrokerReconfigurationTest.scala

```diff
@@ -1128,32 +1125,38 @@ class DynamicBrokerReconfigurationTest extends ZooKeeperTestHarness with SaslSet
 
   private class ProducerThread(clientId: String, retries: Int) extends ShutdownableThread(clientId, isInterruptible = false) {
     private val producer = createProducer(trustStoreFile1, retries, clientId)
+    val sentKeys = new ConcurrentSkipListSet[String]()
```

Comment:
Since the sent keys are just incrementing integers, would it be sufficient to track the last sent? The consumer could verify that received keys are sequential and also keep track of the last received.

Reply:
@hachikuji Thanks for the review. Changed to a map of last sent/received for each partition.